src/commands/convertConnections/ConvertSettingsStep.ts

```diff
@@ -0,0 +1,236 @@
+                        },
+                        {
+                            scopeId: scope,
+                            roleDefinitionId: createRoleId(context.subscriptionId, CommonRoleDefinitions.azureEventHubsDataReceiver),
+                            roleDefinitionName: CommonRoleDefinitions.azureEventHubsDataReceiver.roleName
+                        }
+                    );
+                } else if (connection.name.includes('SERVICEBUS')) {
+                    const serviceBusNamespace = connection.value.split(';')[0].split('/')[2].split('.')[0];
+
+                    context.convertedConnections?.push(
+                        {
+                            name: `${serviceBusNamespace}__fullyQualifiedNamespace`,
+                            value: `${serviceBusNamespace}.servicebus.windows.net`,
+                        },
+                        ...getClientIdAndCredentialProperties(context, serviceBusNamespace)
+                    );
+
+                    const scope = await getScopeHelper(context, serviceBusNamespace, `resourceType eq  'Microsoft.ServiceBus/Namespaces'`);
+                    context.roles?.push(
+                        {
+                            scopeId: scope,
+                            roleDefinitionId: createRoleId(context.subscriptionId, CommonRoleDefinitions.azureServiceBusDataReceiver),
+                            roleDefinitionName: CommonRoleDefinitions.azureServiceBusDataReceiver.roleName
+                        },
+                        {
+                            scopeId: scope,
+                            roleDefinitionId: createRoleId(context.subscriptionId, CommonRoleDefinitions.azureServiceBusDataOwner),
+                            roleDefinitionName: CommonRoleDefinitions.azureServiceBusDataOwner.roleName
+                        }
+                    );
+                }
+            }
+
+            if (!context.localSettingsPath) {
+                const message: string = localize('selectLocalSettings', 'Select the local settings to convert.');
+                context.localSettingsPath = await getLocalSettingsFile(context, message);
+            }
+
+            if (context.local) {
+                const localSettings = await AzExtFsExtra.readJSON<ILocalSettingsJson>(nonNullProp(context, 'localSettingsPath'));
+                if (localSettings.Values) {
+                    for (const connection of context.convertedConnections) {
+                        localSettings.Values[connection.name] = connection.value;
+                        context.activityChildren?.push(
+                            new GenericTreeItem(undefined, {
+                                contextValue: createUniversallyUniqueContextValue(['useExistingResourceGroupInfoItem', activitySuccessContext]),
+                                label: localize('addedSetting', `Add Local setting "${connection.name}"`),
```

Comment:
This isn't localized correctly.

`localize('addedSetting', 'Add Local setting "${0}"', connection.name);`